File: testcases/paper/NestedClasses/NestedClasses.go
package NestedClasses

type A struct {
    i, j, k int
    B struct {
        x, y, z int
    }
}

func (a A) GetDataI() int {
    return a.i
}

func (a A) GetDataJ() int {
    return a.j
}

func (b A) GetDataX() int {
    return b.B.x
}

func (b A) GetDataY() int {
    return b.B.y
}

File: testcases/paper/NestedClasses/NestedClasses.py
class A:
    def __init__(self):
        self.i = 0
        self.j = 0
        self.k = 0

    class B:
        def __init__(self):
            self.x = 0
            self.y = 0
            self.z = 0

        def getdatax(self):
            return self.x

        def getdatay(self):
            return self.y

    def getdatai(self):
        return self.i

    def getdataj(self):
        return self.j
